01/app.js

```diff
@@ -1 +1,10 @@
-console.log('DOM');
\ No newline at end of file
+console.log('DOM');
+
+const className = document.querySelector('.comments__item.comments__item--newest');
+let dataInfo;
+if(className !== null) {
+    dataInfo = className.dataset.info;
+};
```

Comment:
Czy ta część jest potrzebna w tym rozwiązaniu? Zwróć uwag, że nie korzystasz z tego później....

Jak już to tak:
```
const className = document.querySelector('.comments__item.comments__item--newest');
if(className !== null) {
    const elementsWithDataInfo = className .querySelectorAll('[data-info]'); // tutaj był błąd - brakowało zamknięcia nawiasu ]
    console.log(`Ilość znalezionych elementów data-info wynosi ${elementsWithDataInfo.length}`)
};
```